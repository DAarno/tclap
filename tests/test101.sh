#!/bin/sh

../examples/test35 -h -a 5 > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test101.out; then
	exit 0
else
	exit 1
fi
