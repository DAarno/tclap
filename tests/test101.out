
USAGE: 

   ../examples/test35  {-c <int>} [-a <int>] [--] [--version] [-h]


Where: 

   -c <int>,  --c_int <int>
     (OR required)  int arg


   -a <int>,  --a_int <int>
     int arg

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.


   

